terminus/tests/road_test.py

```diff
@@ -66,3 +68,22 @@ def test_length_empty(self):
     def test_length_empty_road(self):
         road = Street.from_points([])
         self.assertEqual(road.length(), 0.0)
+
+        '''
+        this test couldn't be implemented yet:
```

Comment:
That is because you are creating an instance of `RoadNode`, which is an abstract class. You should create either a `RoadIntersectionNode` or a `RoadSimpleNode` for the test to work. Maybe it would be a good time to review a bit on OOP and python (e.g. https://python.swaroopch.com/oop.html and http://anandology.com/python-practice-book/object_oriented_programming.html) and we can later talk a bit about abstract classes.